sql/api/src/main/scala/org/apache/spark/sql/catalyst/trees/QueryContexts.scala

```diff
@@ -134,7 +134,9 @@ case class SQLQueryContext(
   override def callSite: String = throw SparkUnsupportedOperationException()
 }
 
-case class DataFrameQueryContext(stackTrace: Seq[StackTraceElement]) extends QueryContext {
+case class DataFrameQueryContext(
+    stackTrace: Seq[StackTraceElement],
+    pysparkLoggingInfo: Option[java.util.Map[String, String]]) extends QueryContext {
```

Comment:
why is it a map instead of `(String, String)`?

Reply:
Because we're currently using `dict` for PySpark logging info and it internally converted into `Map`:

```
            logging_info = {
                "fragment": name,
                "callSite": f"{frame_info.filename}:{frame_info.lineno}",
            }
```

Maybe is there happen to any reason that we should use `Option[(String, String)]`??

Reply:
it's more precise to use two string instead of a map.

Reply:
Got it. let me update it. Thanks for the suggestion!